container-toolkit/install-guide.md

```diff
@@ -4,22 +4,23 @@
 
 # Installing the NVIDIA Container Toolkit
 
+
 ## Installation
 
 (pre-requisites)=
 
 ### Prerequisites
 
-Install the NVIDIA GPU driver for your Linux distribution.
-NVIDIA recommends installing the driver by using the package manager for your distribution.
+1. Read [this section](./supported-platforms.md) about platform support.
 
+2. Install the NVIDIA GPU driver for your Linux distribution.
+NVIDIA recommends installing the driver by using the package manager for your distribution.
 For information about installing the driver with a package manager, refer to
 the [_NVIDIA Driver Installation Quickstart Guide_](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html).
+Alternatively, you can install the driver by [downloading](https://www.nvidia.com/en-us/drivers/) a `.run` installer.
 
-Alternatively, you can install the driver by downloading a `.run` installer.
-Refer to the NVIDIA [Official Drivers](https://www.nvidia.com/Download/index.aspx?lang=en-us) page.
 
-### Installing with Apt
+### Ubuntu and Debian
```

Comment:
I'm happy with this change for ubuntu and debian since I think this covers most users of `apt`. My concern is that using the same construct for RPM-based distributions where there are other prominent ones that are not listed. (*Rocky* comes to mind). This will definitely lead to GitHub issues along the lines of "RockyLinux support when?". At present my response is to link the "Installing with Yum or Dnf" section, but I can see more questions being raised if I link a section that's titled "RHEL/CentOS, Fedora, Amazon Linux".

Reply:
yes. note: added a commit after working on this with Evan